plugins/modules/azure_rm_openshiftmanagedclusterversion_info.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2020 Haiyuan Zhang <haiyzhan@micosoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_openshiftmanagedclusterversion_info
+version_added: '2.5.0'
+short_description: Fetch available versions of Azure Red Hat OpenShift Managed Cluster
+description:
+    - fetch available version of Azure Red Hat OpenShift Managed Cluster instance.
+options:
+    location:
+        description:
+            - List install versions available for the defined region.
+        required: true
+        type: str
+extends_documentation_fragment:
+    - azure.azcollection.azure
+author:
+    - Maxim Babushkin (@maxbab)
+'''
+
+EXAMPLES = '''
+- name: Obtain openshift versions for ARO cluster
+  azure_rm_openshiftmanagedclusterversion_info:
+    location: centralus
+  register: ocp_versions
```

Comment:
Lines 34~38 are not necessary and can be deleted! Thank you very much!

Reply:
Removed.